bcda-site-static/_includes/copy_snippet.md

```diff
@@ -0,0 +1,27 @@
+{% assign code = include.code %}
+
+<div class="highlight"><pre class="highlight"><code>{{code}}
+</code></pre></div>
+
+{% assign nanosecond = "now" | date: "%N" %}
+<input type="text" id="code{{ nanosecond }}" style="position: absolute;left: -1000px;" value="{{ code }}"/>
+<div class="button" on>
+    <p>
+        <a href="javascript:void()" onclick="copyText{{ nanosecond }}()" id="copybutton{{ nanosecond }}">
```

Comment:
just looked it up, is this supposed to be void(0)?

Reply:
you are right.  I fixed it